docker-compose.yml

```diff
@@ -287,4 +287,10 @@ services:
     image: kataru/content-did-driver:latest
     ports:
       - "8148:8888"
-
+  driver-did-btco:
+    image: brianrichter/did-btco:latest
```

Comment:
Only tag available is `latest`. Ideally, please tag and push an image to a Docker registry with a semantic version number.